api/v1/multiclusterengine_methods.go

```diff
@@ -98,6 +99,7 @@ var MCEComponents = []string{
 	ClusterAPI,
 	ClusterAPIProviderAWS,
 	ClusterAPIProviderMetalPreview,
+	ClusterAPIProviderOA,
 	ClusterAPIProviderOAPreview,
```

Comment:
I believe we can remove the `ClusterAPIProviderOAPreview` entry here since we will be using the GA version.